microservice_c/main.py

```diff
@@ -0,0 +1,13 @@
+import os
+from flask import Flask, jsonify
+app = Flask(__name__)
+
+port = int(os.getenv('LISTEN_PORT', '5000'))
+exit_code = int(os.getenv('EXIT_CODE', '500'))
+
+@app.route('/')
+def hello_world():
+    return jsonify(message="cause i am bad guy"), exit_code
```

Comment:
• When the test script sends an http request to microservice_c, it should fail with a 500 
response code.